MOBILE_CLIENT/COMMON/_POTENTIALLY/SECRETS_REVIEW/meta.json

```diff
@@ -3,7 +3,6 @@
   "short_description": "Passwords, Tokens or other sensitive information are stored in the application.",
   "references": {
     "API Security Best Practices": "https://developers.google.com/maps/api-security-best-practices",
-    "How to secure your Google Maps API Key": "https://www.storemapper.com/support/knowledge-base/how-to-secure-your-google-maps-api-key/",
```

Comment:
I don't understand why this change?

Reply:
Some links are broken, causing the unittest to fail